prow/hook/server.go

```diff
@@ -36,6 +39,20 @@ type Server struct {
 	ConfigAgent *config.Agent
 	HMACSecret  []byte
 	Metrics     *Metrics
+
+	// c is an http client used for dispatching events
+	// to external plugin services.
+	c *http.Client
```

Comment:
nit question: would it work just as well to say `c http.Client` and use the zero value? It looks like then we wouldn't need to have the `NewServer` stuff.

Reply:
should work fine, done